src/test/java/com/example/petinnoflightandairlines/service/AirportServiceIT.java

```diff
@@ -0,0 +1,228 @@
+package com.example.petinnoflightandairlines.service;
+
+import com.example.petinnoflightandairlines.dto.AirportDTO;
+import com.example.petinnoflightandairlines.mapper.AirportMapper;
+import com.example.petinnoflightandairlines.model.Airport;
+import com.example.petinnoflightandairlines.repository.AirportRepository;
+import jakarta.validation.ConstraintViolationException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@Slf4j
+@Transactional
+@SpringBootTest
+@RequiredArgsConstructor
+class AirportServiceIT {
+
+    @Autowired
+    private AirportService airportService;
+
+    @Autowired
+    private AirportRepository airportRepository;
+
+    @Autowired
+    private AirportMapper airportMapper;
+
+    private static final Integer MAX_COUNT_OF_SYNC_FLIGHTS = 4;
+    private static final String NAME = "some_name";
+    private static final String AIRPORT_IATA = "333";
+    private static final String AIRPORT_ICAO = "3334";
+    private static final String LOCATION = "some location";
+
+    @BeforeEach
+    void setUp() {
+        airportService = new AirportServiceImpl(airportRepository, airportMapper);
+    }
+
+    @Test
+    void ifAllGood_thenAddAirport() {
```

Comment:
the better to use the following test name pattern:
test_<method_name>_should<which behavior we test>